src/Microsoft.SqlTools.Serialization/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.SqlTools.Hosting.Utility;
+using Microsoft.SqlTools.ServiceLayer.SqlContext;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.Serialization
+{
+    /// <summary>
+    /// Main application class for Serialization Service Host executable
+    /// </summary>
+    internal class Program
+    {
+        /// <summary>
+        /// Main entry point into the Serialization Service Host
+        /// </summary>
+        internal static void Main(string[] args)
+        {
+            try
+            {
+                // read command-line arguments
+                CommandOptions commandOptions = new CommandOptions(args, "Microsoft.SqlTools.Serialization.exe");
```

Comment:
why the name of the exe for serialization is different than other two? for credentials it's "MicrosoftSqlToolsCredentials.exe"

Reply:
That's the name of the exe generated when I run 'dotnet build -r win7-x64' from src/Microsoft.SqlTools.Serialization. Yeah, it would be ideal if the generated name was the same format. Do you know how the exe name is chosen? I tried doing a search for 'Microsoft.SqlTools.Serialization' but couldn't find anything relevant

Reply:
So I'll change the name to "SqlSerializationService.exe" based off of a previous suggestion by Kevin here: https://github.com/Microsoft/sqltoolsservice/pull/442/files/25ff42794b3730543210e8ebf0318f93bf8cedea#diff-6089a6649676ec48e22c0ad42a2e8834